TaboolaSDKExampleSwift/TaboolaSDKExampleSwift/SDK Classic/CollectionView/ClassicCollectionViewManagedByTaboolaController.swift

```diff
@@ -22,23 +22,21 @@ class ClassicCollectionViewManagedByTaboolaController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
                
-        classicPage = TBLClassicPage.init(pageType: "article", pageUrl: "http://www.example.com", delegate: self, scrollView: self.collectionView)
-        classicPage?.pageExtraProperties = ["key":"true"]
+        classicPage = TBLClassicPage.init(pageType: Constants.pageTypeArticle, pageUrl: Constants.pageUrl, delegate: self, scrollView: self.collectionView)
+        classicPage?.pageExtraProperties = ["key":"true"] // example for pageExtraProperties
```

Comment:
Maybe it's worth emphasizing that the key "key" should be replaced with values given from Taboola representative in relevant cases.